dist/index.html

```diff
@@ -3,54 +3,30 @@
   <body>
     <div id="app">
       Humble Bundle
-      <div id="navbar">
-        Navbar
-      </div>
-      <div id="bundleNav">
-        bundleNav
-      </div>
       <div id="openingDescription">
-        Opening Description
+        Opening Description Not Loading
       </div>
       <div id="Tiers">
-        Tier 1
-        <div id="Item">
-          Item 1
-          <div id="sysRequirements">
-            System Requirements 1
-          </div>
-          <div id="reviews">
-            Reviews 1
-          </div>
-        </div>
-      </div>
-      <div id="bonus">
-        Bonus
-      </div>
-      <div id="charity">
-        Charity
-      </div>
-      <div id="contributors">
-        Contributors
-      </div>
-      <div id="order">
-        Order Summary
+        Tier Not Loading
       </div>
-      <div id="closingDescription">
-        Closing Description
+      <div id="bonuses">
+        Bonus Not Loading
       </div>
-      <div id="footer">
-        Footer
+      <div id="individual-items">
+        Individual Item Not Loading
       </div>
     </div>
     <script src="http://localhost:3101/static/js/bundle.js"></script>
+    <script src="http://localhost:3031/bundle.js"></script>
+    <script src="http://localhost:3200/bundle.js"></script>
+    <script src="http://localhost:3663/bundle.js"></script>
   </body>
 </html>
 
 <!--
 
   The above is not meant to be final.
-  It includes all servers and static sections.
+  It includes services which are currently available.
 
```

Comment:
I think the idea is to only push code that you think is final,  even if later on necessity requires change. Anything that you consciously realize at the time of push will require involved edits should not be pushed to the master but be held in a branch for editing. Especially if the code you are pushing to master, breaks or may break your functionality. 

(Meaning...as long as it works and is not breaking your code, maybe don't frame it with a comment that says its not final. To-may-toe, to-mah-toe.)

Reply:
README should have details useful for anyone trying to deploy the current version of a service, so I tend towards putting details on what may not be working as expected so that it's known instead of someone trying to deploy it thinking it's a bug.